### ISSUE TITLE
install command saves files into new parent directory instead of existing font directory

### ISSUE THREAD

**Issue by bpringe** [paulyoung/fontello-cli#47]
Thanks for making this tool. I'm wondering if there is a way to make the install command replace the contents in the config directory with the downloaded contents, rather than creating a new parent font directory.

## Actual result
If I had my config, css, etc., for my font saved in `./assets/my-font` then run the install command, such as `npx fontello-cli install --config ./assets/my-font/config.json`, the CLI downloads the font and places the files in a new directory in the current directory, called something like `fontello-abcd`, at the same level of the `assets` directory.

## Expected/desired result
I'd like the install command to replace the contents of `./assets/my-font` with the newly downloaded font files, instead of adding them to a new directory.

**Comment by dlackty** [comment]
Have you tried `--css` and `--font` options? Say for example:

```bash
npx fontello-cli install --config ./assets/my-font/config.json \
                         --font ./assets/my-font/font \
                         --css ./assets/my-font/css
```

**Comment by bpringe** [comment]
Yes, I've tried those options, but it seemed to only update the files in the `font` and `css` directories. I'd like it to update all the files, including `config.json` and `demo.html`.

**Comment by webocoders** [comment]
+
